blocks/editable/style.scss

```diff
@@ -3,6 +3,10 @@
 }
 
 .blocks-editable__tinymce {
+	> p {
+		min-height: 28px;
```

Comment:
Two thoughts:

- We can calculate this as `min-height: $editor-font-size * $editor-line-height;` (with `round` or `floor` maybe?) which would help maintainability if those values were to ever change. 
- Should we change the selector target the empty paragraph `> p:empty` ?